create_test_instance.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2016 Google LLC All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from google.cloud.spanner_v1 import Client
+
+if os.getenv(
+    "SPANNER_EMULATOR_HOST"
+):  # otherwise instance will be created by parallelize_tests
+    project = os.getenv(
+        "GOOGLE_CLOUD_PROJECT",
+        os.getenv("PROJECT_ID", "emulator-test-project"),
+    )
+
+    client = Client(project=project)
+
+    config = f"{client.project_name}/instanceConfigs/regional-us-central1"
```

Comment:
I'm surprised you need the instance config, what does https://github.com/googleapis/python-spanner-django/pull/569/commits/720ef6a77d75a2a2a622e298eea42eb78d2efe6d do?

Reply:
It doesn't want to create an instance without minimal configurations, so we're usually creaing instances this way